astroquery/mast/tests/test_mast_remote.py

```diff
@@ -266,6 +255,8 @@ def test_observations_filter_products(self):
         assert isinstance(result, Table)
         assert len(result) == sum(products['productType'] == "SCIENCE")
 
+    # test downloads 150+ files, 50MB+, TODO: revise OBSID to query only a few, small files for download
```

Comment:
The code snippet from this test takes an awfully long time, I've cancelled it after many minutes, when it already downloaded 150+ files, and 50MB+. I'll open a separate issue as a reminder that the test needs to be fixed in a way that only a very small data download should happen at test time.